src/main/java/jenkins/plugins/http_request/HttpRequest.java

```diff
@@ -281,19 +323,31 @@ private HttpRequestBase getHttpRequestBase(PrintStream logger, RequestAction req
         return httpRequestBase;
     }
 
-    private FilePath getOutputFilePath(AbstractBuild<?, ?> build) {
+    private FilePath getOutputFilePath(FilePath workspace, PrintStream logger) {
+        if (workspace == null) {
+            logger.println("Workspace is null, will not save to the outputFile. To get a workspace, execute inside a Workflow node block.");
+            return null;
+        }
```

Comment:
Unnecessary. You would not get this far if you were not within `node`; the `step` step would fail before even considering its `delegate`.